src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -300,28 +305,19 @@ bool java_bytecode_languaget::generate_support_functions(
 /// instantiated (or evidence that an object of that type exists before the main
 /// function is entered, such as being passed as a parameter).
 /// \par parameters: `symbol_table`: global symbol table
-/// `lazy_methods`: map from method names to relevant symbol and parsed-method
-///   objects.
+/// `method_bytecode`: map from method names to relevant symbol and
+/// parsed-method objects.
 /// \return Elaborates lazily-converted methods that may be reachable starting
 ///   from the main entry point (usually provided with the --function command-
 ///   line option) (side-effect on the symbol_table). Returns false on success.
 bool java_bytecode_languaget::do_ci_lazy_method_conversion(
   symbol_tablet &symbol_table,
-  lazy_methodst &lazy_methods)
+  method_bytecodet &method_bytecode)
 {
-  const auto method_converter=[&](
-    const symbolt &symbol,
-    const java_bytecode_parse_treet::methodt &method,
-    ci_lazy_methods_neededt new_lazy_methods)
-  {
-    java_bytecode_convert_method(
-      symbol,
-      method,
-      symbol_table,
-      get_message_handler(),
-      max_user_array_length,
-      safe_pointer<ci_lazy_methods_neededt>::create_non_null(&new_lazy_methods),
-      string_preprocess);
+  const method_convertert method_converter = [this, &symbol_table](
+    const irep_idt &function_id, ci_lazy_methods_neededt lazy_methods_needed) {
```

Comment:
cpp_lint will complain that the brace should be on a new line

Reply:
But clang-format forces this style. The linter will have to cope.

Reply:
Please take one of three actions:
1. Add `NOLINT(whitespace/braces)`
2. Ignore `clang-format` and follow the linter's opinion
3. Fix the linter

Reply:
you can add `// clang-format off` before this line to deactivate it...